source/SIL.AppBuilder.Portal.Frontend/src/ui/components/product-icon/index.tsx

```diff
@@ -11,31 +11,44 @@ const colorStyles = {
   html: { color: '#f5a623' }
 };
 
+const styles = (type, selected,size) => {
+  const color = selected ? colorStyles[type] : DEFAULT_COLOR;
+  const fontSize = size ? { fontSize: `${size}px` } : '';
+
+  return {
+    ...color,
+    ...fontSize
+  };
+};
+
 const DEFAULT_COLOR = { color: '#9b9b9b' };
 
 const iconMap = {
-  android: (selected) => <AndriodIcon style={selected ? colorStyles.android : DEFAULT_COLOR} />,
-  html: (selected) => <WebIcon style={selected ? colorStyles.html : DEFAULT_COLOR} />,
+  android: (selected, size) =>
+    <AndriodIcon style={styles('android',selected, size)} />,
```

Comment:
Not that this is a big thing, but shouldn't this be AndroidIcon not AndriodIcon

Reply:
nice catch